src/main/resources/config/application.yml

```diff
@@ -42,6 +65,9 @@ spring:
             discovery:
                 healthCheckPath: /management/health
                 instanceId: ${spring.application.name}:${spring.application.instance-id:${random.value}}
+            config:
+                watch:
+                    enabled: false
     jackson:
         serialization.write_dates_as_timestamps: false
         default-property-inclusion: NON_NULL
```

Comment:
@vkirichenko don't we need to define `NON_NULL` in lower case as `non_null`?

Reply:
changed to non_empty
non_empty also exclude empty collections